piaplib/keyring/clearify.py

```diff
@@ -136,7 +147,7 @@ def getKeyFilePath():
 
 
 @remediation.error_handling
-def makeKeystoreFile(theKey=str(rand.randPW(16)), somePath=None):
+def makeKeystoreFile(theKey=str(str(rand.randPW(16)).replace("%", "%%")), somePath=None):
```

Comment:
This is just a flaw of python, unless using the operator or the format function the '%' should not crash the program. this was a weird fix.